glossary.md

```diff
@@ -1,16 +1,80 @@
 You may also need to check https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md.
 
-* LGTM: "Looks good to me", commonly used to approve a code review.
-* PTAL: Please take a look.
-* RSLGTM: "Rubber-stamp looks good to me". The reviewer approving without doing
+* ABI: [Application Binary Interface](https://en.wikipedia.org/wiki/Application_binary_interface)
+* ASAP: As Soon As Possible.
+* ASYNC: Asynchronous
```

Comment:
Why is `SYNC` left out if `ASYNC` needs explanation?

Reply:
Removed it.